sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -225,11 +238,9 @@ export default defineComponent({
     };
 
     const getOfferingDetails = async (form: FormIOForm, locationId: number) => {
-      const selectedIntensity: OfferingIntensity =
-        formioUtils.getComponentValueByKey(form, OFFERING_INTENSITY_KEY);
       const educationProgramIdFromForm: number =
         formioUtils.getComponentValueByKey(form, PROGRAMS_DROPDOWN_KEY);
-      if (educationProgramIdFromForm && selectedIntensity) {
+      if (educationProgramIdFromForm && offeringIntensity) {
```

Comment:
Similar to this one: https://github.com/bcgov/SIMS/pull/4679/files#r2076083024
Was it kept because the variable is not initialized at first?